tools/tungsten/core/src/com/google/android/filament/tungsten/model/Nodes.kt

```diff
@@ -43,27 +45,63 @@ private val adderNodeCompile = fun(node: Node, compiler: GraphCompiler): Node {
     return node
 }
 
+private val UNLIT_INPUTS = listOf("baseColor", "emissive")
+private val LIT_INPUTS = listOf("baseColor", "metallic", "roughness", "reflectance", "clearCoat",
+        "clearCoatRoughness", "ambientOcclusion", "emissive")
```

Comment:
You're missing `anisotropy`, `anisotropyDirection`, `normal` and `clearCoatNormal`

Reply:
Yeah, I'm still deciding how to handle anisotropy. since it's -1 to 1, the `JColorChooser` can't handle it. As for the others, since they represent a new semantic of "direction", it might be better to have an alternative property editor to specify a direction.